maintenance/wikia/getDatabase.php

```diff
@@ -75,14 +77,10 @@
 			exit;
 		}
 		/* Map cluster numbers to letters
-		 * 1->A, 2->B
+		 * 1->a, 2->b
 		 * chr(65)=='A' */
 		$clusterLetter = chr( 64 + $clusterNumberParam );
-		$databaseDirectory = "database_{$clusterLetter}";
-		if ( $clusterNumberParam >= 6 ) {
-			// Way of combining s3_bucket name changed from cluster 6
-			$databaseDirectory = 'database-' . strtolower( $clusterLetter );
-		}
+		$databaseDirectory = 'database-' . strtolower( $clusterLetter );
```

Comment:
The directory names were different for different clusters (some use - and some use _)